tc/tc_util.c

```diff
@@ -196,6 +197,7 @@ int parse_percent_rate(char *rate, const char *str, char *dev)
 	long max_rate_bits;
 	int ret;
 	double perc, rate_bits;
+	char *char_perc, *p;
```

Comment:
Let's change `char_perc` to `str_perc` to more accurately reflect what it represents.